packages/base-controller/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.2.1]
+### Changed
+- There are no consumer-facing changes to this package. This version is a part of a synchronized release across all packages in our monorepo.
```

Comment:
Should we do this for controller-utils as well? It's the only package not included in this release at the moment

Reply:
The create-release-branch tool we use to create releases highlights every package with changes when you go to create a new release, with the expectation that the author of the release PR would review each of these packages to determine whether they need to be released (and if so, which version they should have).

This release is intended to clear the slate, so that future releases are easier to manage.

This situation will be less of a problem soon (we have some improvements to create-release-branch that we'll be working on soon to make it a bit smarter about releasing one package without the others safely). But that's a week or two out at best.

Reply:
That's true. We also changed how packages refer to other packages within the monorepo. Now they use static versions always instead of dynamic versions which get resolved at publish-time. So we also want to publish every package regardless of whether there were actual changes to ensure that all packages have up-to-date references (and expose any issues that we may see as a result of this change sooner rather than later).